testing/test_assertion.py

```diff
@@ -797,8 +797,8 @@ def test_recursive_dataclasses(self, testdir):
                 "*Differing attributes:*",
                 "*field_b: ComplexDataObject2(*SimpleDataObject(field_a=2, field_b='c')) != ComplexDataObject2(*SimpleDataObject(field_a=3, field_b='c'))*",  # noqa
                 "*Drill down into differing attribute field_b:*",
-                "*Omitting 1 identical items, use -vv to show*",
                 "*Differing attributes:*",
+                "*field_a: ComplexDataObject*",
```

Comment:
Because the indentation is an important part of the output, the tests should include them in the test, i.e. drop the `*` wildcards at the beginnings. Otherwise the tests accept any indentation, and it is not verified to be correct.